src/stars.css

```diff
@@ -0,0 +1,91 @@
+.tutto {
+  height: 160px;
+  width: 140px;
+}
```

Comment:
it's a block. can't set external styles for blocks

Reply:
thanks. 
it's updated and we got rid of .tutto